src/appmixer/redmine/core/IssueCreate/component.json

```diff
@@ -55,34 +57,44 @@
                     "issue|fixed_version_id": {
                         "title": "FixedVersionId",
                         "type": "integer",
-                        "description": "ID of the Target Versions (previously called \"Fixed Version\" and still referred to as such in the API)",
+                        "example": 1,
+                        "description": "ID of the Target Versions (previously called \"Fixed Version\" and still referred to as such in the API).",
                         "path": "issue.fixed_version_id"
                     },
                     "issue|assigned_to_id": {
                         "title": "AssignedToId",
                         "type": "integer",
-                        "description": "ID of the user to assign the issue to (currently no mechanism to assign by name)",
+                        "example": 34,
+                        "description": "ID of the user to assign the issue to (currently no mechanism to assign by name).",
                         "path": "issue.assigned_to_id"
                     },
                     "issue|parent_issue_id": {
                         "title": "ParentIssueId",
                         "type": "integer",
-                        "description": "ID of the parent issue",
+                        "example": 43,
+                        "description": "ID of the parent issue.",
                         "path": "issue.parent_issue_id"
                     },
                     "issue|custom_fields": {
                         "title": "CustomFields",
-                        "type": "array",
+                        "type": "string",
                         "path": "issue.custom_fields",
+                        "originalType": "array",
                         "items": {
```

Comment:
@jirihofman Since you changed the type from an array to a string, do we still need to keep these items?

Reply:
We don't need `items` anymore but that would need to be handled in the generator. As of now it keeps many other properties there if they are in openAPI specs even though Appmixer doesn't use them.

Definitely a good idea to start removing them in the generator so that our `component.json` files are smaller.